### PR TITLE
Add Auto-Complete Functionality to Parent Facility Field - Client Walk In Form

### PR DESCRIPTION
**Pull request by semujju**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java [hunk 1]
```diff
@@ -324,4 +324,19 @@ public String uuid() {
             return "afcdfcd8-defe-11eb-ba80-0242ac130004";
         }
     };
+    public static EncounterTypeDescriptor WALKIN_CLIENT_CAPTURE = new EncounterTypeDescriptor() {
+        @Override
+        public String name() {
+            return "Walkin Art Service";
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java]: @semujju Change this to "Emergency ART Service Encounter"

File: api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java [hunk 2]
```diff
@@ -324,4 +324,19 @@ public String uuid() {
             return "afcdfcd8-defe-11eb-ba80-0242ac130004";
         }
     };
+    public static EncounterTypeDescriptor WALKIN_CLIENT_CAPTURE = new EncounterTypeDescriptor() {
```
Comment (ssmusoke) [inline api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java]: @semujju change the constant to EMERGENCY_ART_SERVICE

File: api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java (hunk 1 above)
Comment (semujju) [reply api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java]: Done

File: api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java (hunk 2 above)
Comment (semujju) [reply api/src/main/java/org/openmrs/module/ugandaemr/metadata/core/EncounterTypes.java]: Done

**Comment by ssmusoke** [comment]
@dsemujju the list of facilities includes HCII which are not ART accredited sites

**Comment by ssmusoke** [comment]
@ODORA0 This is additional feedback on this form - not sure if you or @semujju will be working on it 

1. District of Parent Health Facility will need to be a dropdown of districts 
2. Reason for visit - use alphabetical order 
2. Change the label for Other Reason for visit from Other Specify to Other Reason for Visit 
3. Other fields for Reason for Visit and Regimen are misplaced see images 
Screen Shot 2021-09-28 at 10 25 28
Screen Shot 2021-09-28 at 10 25 46
4. EID number is enterable for an adult - tried it with a 31 year old male 
5. Client Category - use alphabetical order
6. Client Category - add a space before the dash between the code and text 
7. Clinical Assessment and Refill - change Other label to Other ART Regimen and it is misplaced 
8. TB status does not look like a drop down 
Screen Shot 2021-09-28 at 10 56 54
9. Form does not save or show validation errors 
Screen Shot 2021-09-28 at 10 59 19

**Pull request merged**